### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by jordansissel**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.1.0
+ - New year rollover should be handled better now when a year is not present in
```
Comment (jsvd) [inline CHANGELOG.md]: Maybe first line could be
* no longer memorize current year at startup, which caused incorrect timestamps after new year

**Comment by suyograo** [comment]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+   the time format. If local time is December, and event time is January, the
+   year will be set to next year. Similar for if local time is January and
+   Event time is December, the year will be set to the previous year. This
+   should help keep times correct in the upcoming year rollover. (#33, #4)
```
Comment (jsvd) [inline CHANGELOG.md]: Support reading timezone from event (#31)

**Comment by jsvd** [comment]
Suggested 2 additions, otherwise LGTM

**Comment by colinsurprenant** [comment]
@jordansissel since #45 has been merged without a version bump, I suggest include add #45 changelog here to version 2.1.0. It could be something like:
```
- use Event#tag, relax specs for Java Event, code cleanups
```

Let me know what you think, otherwise I'll create a version bump PR just for #45.

**Comment by jordansissel** [comment]
@colinsurprenant will include your changelog suggestion. thanks! :)

**Pull request closed without merging**